dts/arm/nxp/nxp_rt.dtsi

```diff
@@ -341,9 +347,10 @@
 			reg = <0x402E0000 0x200>;
 			interrupts = <113 1>;
 			interrupt-names = "usb_otg";
+			clocks = <&sysclk>;
 			num-bidir-endpoints = <8>;
 			maximum-speed = "full-speed";
-			status = "disabled";
+			status = "ok";
```

Comment:
Good question. Since @pizi-nordic is not here right now, @jfischer-phytec-iot do you know the reason?

Reply:
I checked with @pizi-nordic. This is required because with the USB disabled the clock information is not available and it is required during SoC setup

Reply:
Where is that soc code?  Seems like its missing an ifdef for USB driver being enabled or something.

Reply:
Here: https://github.com/zephyrproject-rtos/zephyr/pull/16958/files#diff-be3b6b842e8552442c99fc8abcafeafd

Reply:
I don't understand why this is needed. Can you point me to a test or sample that fails sanitycheck without this change? The mimxrt10{15,20,60} boards don't yet enable usb in their board.yaml files, so sanitycheck should not be trying to build `samples/subsys/usb` for them.

Reply:
It failed on mimxrt1050_evk after applying https://github.com/zephyrproject-rtos/zephyr/pull/16958/files#diff-be3b6b842e8552442c99fc8abcafeafd.

But I see that our scripts started to generate clock information for disabled nodes, so the problem no longer exists.

I have removed the change of status from this PR.